packages/react-animation/src/useFlip/useFlip.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import * as stories from './useFlip.stories';
+
+<Meta title="components/useFlip" of={stories} />
+
+# useFlip
```

Comment:
It's unclear from the (non-existing) api docs:

- what I can pass as first parameter (can it be both an element, or also a ref around an element?)
- what I can pass as second parameter (some gsap flip props?)
- If the second param is optional or required.